shell/platform/windows/windows_registry.h

```diff
@@ -0,0 +1,33 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_WINDOWS_WINDOWS_REGISTRY_H_
+#define FLUTTER_SHELL_PLATFORM_WINDOWS_WINDOWS_REGISTRY_H_
+
+#include <Windows.h>
+
+#include "flutter/fml/macros.h"
+
+namespace flutter {
+
+class WindowsRegistry {
```

Comment:
Add a class doc, specifically covering the 'why' for this class (that it makes mocking registry responses easier in unit tests).